.distignore

```diff
@@ -1,10 +1,12 @@
 /.github
 /.git
 /.wordpress-org
-/node_modules
 /.sources
 /bin
 /docs
+/node_modules
+/packages
+/patches
```

Comment:
This directory was removed by @Aljullu in #6465 - is it worth keeping this around?

Reply:
Never mind, it came back because the PR got reverted!